Libplanet.Tests/Serialization/Bencodex/BencodexFormatterTest.cs

```diff
@@ -115,6 +122,7 @@ protected Person(SerializationInfo info, StreamingContext context)
                 .ToList();
             Secret = info.GetString("Secret");
             IsStudent = info.GetBoolean("IsStudent");
+            Assets = (BigInteger)info.GetValue("Assets", typeof(BigInteger));
```

Comment:
We already have [`GetValue<T>()`](https://github.com/planetarium/libplanet/blob/bd4345abde3b31d1f5cc4504da63a7601614d6fe/Libplanet/Serialization/SerializationInfoExtension.cs#L8), so you can use it.